FWCore/MessageService/test/UnitTestClient_H.cc

```diff
@@ -14,8 +14,17 @@ void
                            , edm::EventSetup const & /*unused*/
                               )
 {
+#ifdef EDM_ML_DEBUG
```

Comment:
At the top of the file set
```
#define EDM_ML_DEBUG
```
So the test will always give the same output. Otherwise the unit test will fail when compiled without debug set.

This should also be done for the other unit test file.